MainRepository/Program.cs

```diff
@@ -1,7 +1,28 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿//Program to find the frequency of alphabets in a dictionary.

```

Comment:
```suggestion
﻿// Program to find the frequency of alphabets in a dictionary.

```
Leave a space after // in comments